CMakeLists.txt

```
@@ -0,0 +1,24 @@
+cmake_minimum_required(VERSION 3.26.4 FATAL_ERROR)
+
+project(
+  nvjitlink
+  VERSION 0.1.0
+  LANGUAGES C CXX
```

Do we need C or is it just C++ code?
```suggestion
  LANGUAGES CXX
```